pages/index.jsx

```diff
@@ -0,0 +1,58 @@
+import GameList from "@/components/GameList";
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function HomePage({ games }) {
+  return (
+    <>
+      <StyledMain>
+        <h1>Game Flow</h1>
+        <StyledPageHeadline>Game List</StyledPageHeadline>
+        <section>
+          <StyledListLabel>In progress</StyledListLabel>
+          <GameList progressLabel={"In Progress"} games={games} />
+        </section>
+        <section>
+          <StyledListLabel>Planned</StyledListLabel>
+          <GameList progressLabel={"Planned"} games={games} />
+        </section>
+        <section>
+          <StyledListLabel>Completed</StyledListLabel>
+          <GameList progressLabel={"Completed"} games={games} />
+        </section>
+      </StyledMain>
+      <StyledAddLink href="/add-game/">+</StyledAddLink>
+    </>
+  );
+}
+
+const StyledAddLink = styled(Link)`
```

Comment:
I would adjust the styling of this link slightly.

You seem to try to center the text, but it appears not to be working as intended.
The issue might be connected to the `font-size`, `width` and `height` all having the same value.